pkg/install/install.go

```diff
@@ -543,6 +557,41 @@ func replaceImage(image string, objs []kclient.Object) ([]kclient.Object, error)
 	return objs, nil
 }
 
+func replaceCabundleVolumes(objs []kclient.Object) ([]kclient.Object, error) {
+	for _, obj := range objs {
+		ustr := obj.(*unstructured.Unstructured)
+		if ustr.GetKind() == "Deployment" {
+			containers, _, _ := unstructured.NestedSlice(ustr.Object, "spec", "template", "spec", "containers")
+			for _, container := range containers {
+				container.(map[string]any)["volumeMounts"] = []interface{}{
+					map[string]any{
+						"name":      "cabundle",
+						"mountPath": "/etc/ssl/certs/ca-certificates.crt",
+						"subPath":   "ca-certificates.crt",
+						"readOnly":  true,
+					},
+				}
+			}
+			if err := unstructured.SetNestedSlice(ustr.Object, containers, "spec", "template", "spec", "containers"); err != nil {
+				return nil, err
+			}
+
+			volumes := []interface{}{
+				map[string]any{
+					"name": "cabundle",
```

Comment:
nit: If we end up keeping this constant secret name that we point to, can we create a `const` for the name we're looking for?

Reply:
Changed this to use contants. 👍